notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,949 @@
+{
```

Comment:
<p>Is there something wrong with the last item? It is indented and a "10" when it should be a "9"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>can you explain? i only see a 9..</p>

Reply:
<p>it must be a bug in reviewNB then. don't worry about it</p><p> </p>